plutus-ledger-api/src/PlutusLedgerApi/V1/Data/Value.hs

```diff
@@ -317,27 +317,27 @@ instance HasBlueprintSchema Value referencedTypes where
         , maxItems = Nothing
         }
 
-instance Haskell.Eq Value where
-  (==) = eq
-
-instance Eq Value where
+instance PlutusTx.Eq Value where
   {-# INLINEABLE (==) #-}
   (==) = eq
 
-instance Haskell.Semigroup Value where
-  (<>) = unionWith (+)
+instance Haskell.Eq Value where
```

Comment:
@zliu41 Some cleanup along the way, and use a single implementation between PlutusTx && Haskell